### PR TITLE
***UI POLISH***

### PR DESCRIPTION
**Pull request by kaysabelle**
# General polishing of UI:
- streamline index.css
- remove styling from index.html and move it to index.css (good practice for web dev)
- remove `Get I/O Stream` button
- hide user control buttons (bluetooth toggle, barcode scanner, get data) while deleting database rows!

also the awesome RadLib logo is BACK BY POPULAR DEMAND!!!!!!! :smile: 
alt text

### CODE REVIEW

File: radlibDemoApp/www/css/index.css [hunk 1]
```diff
@@ -79,6 +48,12 @@ div.headerDiv {
    text-align: center;
 }
 
+#logo {
```
Comment (MinickLS) [inline radlibDemoApp/www/css/index.css:51]: woo!

File: radlibDemoApp/www/index.html [hunk 2]
```diff
@@ -21,47 +21,47 @@
     <body onload="initialize_all()">
       <div class="topcoat-navigation-bar">
          <div class="topcoat-navigation-bar__item left quarter">
-            <a id="slide-menu-button" class="topcoat-icon-button--quiet slide-menu-button" style="margin-top:5px;" onclick="toggleMenu()">
-              <span class="topcoat-icon topcoat-icon--menu-stack"></span>
-             </a>
+            <a id="slide-menu-button" class="hamburger topcoat-icon-button--quiet slide-menu-button" onclick="toggleMenu()">
```
Comment (jeslin) [inline radlibDemoApp/www/index.html:24]: dem hamburgers though

**Comment by jeslin** [comment]
GET IT?! POLISH FOR THE TOP COAT?!

**Comment by jeslin** [comment]
But other than that, it looks beautiful

**Comment by kaysabelle** [comment]
anyone wanna merge this? :cry:

**Comment by jeslin** [comment]
The lonely pull request lololol I got you

**Comment by kaysabelle** [comment]
Thanks Jeslin!

**Pull request merged**